### ISSUE TITLE
New image lightbox lost image size & resolution metadata

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-web#17014]
The new lightbox is lightyears ahead of the old one, thanks @niquewoodhouse & @SimonBrandner - but I miss my stats for nerds :'(

**Comment by niquewoodhouse** [comment]
@t3chguy this probably seems really obvious to you but would you mind just explaining why this is important/when its important to know and how important it is compared to other things on the screen.   

I'm just struggling to see where it fits in the hierarchy, and not seeing it in competitor apps made me think its not actually helpful.  Any insight would be really welcome, thanks :)

**Comment by t3chguy** [comment]
Fair point about it not being in competitors, I find it important to save me having to download it or inspecting the event to find that out, e.g if I am considering using something as a wallpaper/screensaver I'd need to know the resolution to see whether its good enough

It isn't super important, I think it could be in the subscript text with the date/time at 
![image](https://user-images.githubusercontent.com/2403652/115391175-12c6ef80-a1d7-11eb-9eb3-31550b0eb164.png)


e.g

`$DATE $TIME - $INFO`

Maybe the size could just be on the download tooltip, I don't think they need to be super accessible

![image](https://user-images.githubusercontent.com/2403652/115391388-51f54080-a1d7-11eb-851b-2418c4240d33.png)

**Comment by waclaw66** [comment]
@SimonBrandner Buttons are show over those in background, it looks messy...

![obrazek](https://user-images.githubusercontent.com/15554561/115659764-5470ab00-a33b-11eb-84de-222262d14397.png)

**Comment by SimonBrandner** [comment]
@waclaw66, I've heard that this could happen but there wasn't a clear reproduction case. Could you please open a new issue and add some steps to reproduce it? I'll try to look into it

**Comment by robintown** [comment]
We now show the file size, but still not the resolution
